src/components/Paginator/Paginator.tsx

```diff
@@ -11,7 +11,10 @@ import {
 	SingleSelectDumb as SingleSelect,
 	ISingleSelectState,
 } from '../SingleSelect/SingleSelect';
-import TextField, { ITextFieldProps } from '../TextField/TextField';
+import TextField, {
+	ITextFieldProps,
```

Comment:
nitpick: are both ITextFieldProps/ITextFieldPropsWithPassThroughs still used? else can be removed from the imports

Reply:
Removed with ITextFieldPropsWithPassThroughs